src/yea_wandb/mock_server.py

```diff
@@ -499,7 +499,7 @@ def graphql():
                     }
                 }
             )
-        if "__type" in body["query"]:         
+        if "query Introspect" in body["query"]:
```

Comment:
can you go even more specific.  otherwise the next person to add introspection will be confused that their test fails with this very strange value of returning the specific hardcoded type.

Reply:
yes i can restore to `__type(name: ....`